WebRoot/js/zimbraMail/prefs/view/ZmPreferencesPage.js

```diff
@@ -674,6 +674,9 @@ function(parentIdOrElem, text, width, listener, id) {
 	button.setText(text);
 	button.addSelectionListener(listener);
 	this._replaceControlElement(parentIdOrElem, button);
+	if(className) {
```

Comment:
As stated above, if the inner functions call stack of `DwtControlInstance.addClassName` are taking care of condition when no classname is passed, adding a if check is  redundant here, I think.

`Dwt.delClass` is called internally in the subsequent function call stack when ever `DwtControlInstance.addClassName` is called and I referred them above for reference.